protocols/gossipsub/src/behaviour.rs

```diff
@@ -2143,7 +2135,7 @@ where
                     topic=%topic_hash,
                     "HEARTBEAT: Mesh low. Topic contains: {} needs: {}",
                     peers.len(),
-                    self.config.mesh_n_low()
+                    self.config.mesh_n()
```

Comment:
But isn't `mesh_n_low` the minimum that it needs?

Reply:
not in this case, we target `mesh_n`, see two lines bellow:
https://github.com/libp2p/rust-libp2p/blob/46874bb744c67f651d29fa1337dbac1dce5607f2/protocols/gossipsub/src/behaviour.rs#L2140-L2141

which conforms to the [spec](https://github.com/libp2p/specs/blob/master/pubsub/gossipsub/gossipsub-v1.0.md#mesh-maintenanc)